### DISCUSSION TITLE
auto play video with possibility to start audio later

### DISCUSSION THREAD

**Discussion by oktober29** [deckerst/aves#1284] [Ideas]
I'd like to have an option to:
- auto play video without audio
- when video is playing, have a button to unmute, or start playing video (without audio) with a triangle over it to fully play (with audio).

In this way i see clearly it's a video, without disturbing, but if wanted, we can play with audio in any moment.

However, what a great app!!!

**Comment by deckerst** [comment]
> auto play video without audio

Already possible, via `Settings` > `Video` > `Playback` > `Auto play` > `Play muted`.

> when video is playing, have a button to unmute, or start playing video (without audio) with a triangle over it to fully play (with audio).

Already possible, via `Settings` > `Viewer` > `Quick actions` > 3rd page of `Available Buttons` has a `Mute` button which allows to mute/unmute.
